lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'twilio_api'
+
+class Order
+  attr_reader :order, :dishes
+
+  def initialize(menu)
+    @order = Hash.new(0)
+    @dishes = menu.dishes
+  end
+
+  def add(food, amount)
+    fail 'Not on the menu!' unless dishes.keys.include?(food)
+    amount.times { @order[food] += 1 }
```

Comment:
This is quite complex. You should extract this out to a private method to make your code more readable.